packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-list-card.tsx

```diff
@@ -182,57 +186,204 @@ export const PropertyTypeRow = ({
   propertyIndex: number;
   onRemove: () => void;
 }) => {
-  const { watch } = useFormContext<EntityTypeEditorForm>();
+  const { control, watch } = useFormContext<EntityTypeEditorForm>();
+  const { update } = useFieldArray({
+    control,
+    name: "properties",
+  });
+
   const propertyTypes = usePropertyTypes();
-  const propertyId = mustBeVersionedUri(
-    watch(`properties.${propertyIndex}.$id`),
-  );
+  const [anchorEl, setAnchorEl] = useState<HTMLDivElement | null>(null);
+  const [multipleValuesMenuOpen, setMultipleValuesMenuOpen] = useState(false);
+
+  const draftProperty = watch(`properties.${propertyIndex}`);
+
+  const { $id, array, minValue, maxValue, required } = draftProperty;
+
+  const propertyId: VersionedUri = mustBeVersionedUri($id);
```

Comment:
`mustBeVersionedUri` returns `VersionedUri` (or it ought to) 

```suggestion
  const propertyId = mustBeVersionedUri($id);
```

Reply:
Removed it in a0904cc0f15b83072f575d8b08b3c985fda5ae54.